config/master.d/master.conf

```diff
@@ -3,7 +3,8 @@ auto_accept: True
 interface: 0.0.0.0
 event_return: mysql
 presence_events: True
-worker_threads: 10
+worker_threads: 16
+timeout: 60
```

Comment:
I am not sure that this is needed - the `timeout` param is badly named - see https://github.com/saltstack/salt/issues/10940#issuecomment-36773465